Fortran.md

```diff
@@ -0,0 +1,21 @@
+Fortran Development
+-------------------
+
+### Skill Set #1: Beginner Usage
+* [An Introduction to Fortran](http://nci.org.au/user-support/training/training-exercises/fortran-programming-basics/)
+
+### Skill Set #2: Advanced Usage
+* [Advanced Fortran Programming](http://nci.org.au/user-support/training/training-exercises/fortran-programming-advanced/)
+* Optimisation
+* Parallelisation
```

Comment:
Do we really want to train them in Parallelisation? I guess it doesn't have to come from us necessarily.

Reply:
The UKMO recommended I use parallelisation for simple array initialisations like `A(:) = 0.0`, and they said that this is the way things are moving as there are more cores per socket, so I guess it's something some people might need to know.